src/gamma_inc.jl

```diff
@@ -943,33 +954,40 @@ function _gamma_inc_inv(a::Float64, p::Float64, q::Float64)
         x0 = gamma_inc_inv_asmall(a, p, q, pcase)
     else    #large a
         haseta = true
-        (x0,fp) = gamma_inc_inv_alarge(a, porq, s)
+        x0, fp = gamma_inc_inv_alarge(a, porq, s)
     end
 
     t = 1
     x = x0
     n = 1
     logabsgam = logabsgamma(a)[1]
-    #Newton-like higher order iteration with upper limit as 15.
+    # Newton-like higher order iteration with upper limit as 15.
     while t > 1.0e-15 && n < 15
         x = x0
         x² = x*x
         if !haseta
             dlnr = (1.0 - a)*log(x) + x + logabsgam
             if dlnr > log(floatmax(Float64)/1000.0)
-                n = 20
+                break
```

Comment:
The Fortran code returns an error code if the Newton method reaches the upper limit `n = 15`. I wonder if one should indicate in the Julia implementation as well if convergence fails.

Reply:
Yeah. I've been thinking a bit about this as well. I'm not sure what the right solution is. It seems like a general issue in Julia that we don't really expose and use error codes. Most often they are either ignored or will cause an exception. Neither seem like a good solution for programs that indend to be reliable. I don't think there is much we can do within the scope of this PR, though.

Reply:
Yes, I was just wondering. I also don't think anything should changed in this PR.